src/release/main.d

```diff
@@ -173,6 +172,40 @@ void main ( string[] params )
 }
 
 
+/*******************************************************************************
+
+    Creates a github release, including the associated milestone link at the top
+
+    Params:
+        con = connection to use
+        repo = repo to use
+        ver = version to release
+        notes = version notes
+
+*******************************************************************************/
+
+void createGithubRelease ( HTTPConnection con, Repository repo, string ver,
+                           string notes )
+{
+    import release.github;
+    import release.gitHelper;
+
+    import octod.api.releases;
+    import std.algorithm;
+    import std.stdio;
+    import std.range;
+
+    auto mstone_list = listMilestones(con, repo).find!(a=>a.title == ver);
+
+    string mstone_link;
+
+    if (!mstone_list.empty)
```

Comment:
Can it really happen? You can't create multiple milestones with the same name and find predicate looks for exact match. I'd simply enforce it.